app/views/application/_reviews.slim

```diff
@@ -30,16 +30,21 @@
         th Reviewer
         th Time
     tbody
-      -sorted_array = reviews.sort {|a,b| b.updated_at <=> a.updated_at }
+      -if GalleryConfig.notebook_title_parse.parsing_enabled
+        -sorted_array = reviews.order(:updated_at).sort {|a,b| a.notebook.title.partition(GalleryConfig.notebook_title_parse.after_character)[2].downcase <=> b.notebook.title.partition(GalleryConfig.notebook_title_parse.after_character)[2].downcase }
+      -else
+        -sorted_array = reviews.order(:updated_at).sort {|a,b| a.notebook.title.downcase <=> b.notebook.title.downcase }
       -sorted_array.each do |review|
         -if !review.notebook.public? && !(@user.admin? || @user.can_edit?(review.notebook))
           -next
         -else
           tr
-            td
-              -if !review.notebook.public? && (@user.admin? || @user.can_edit?(review.notebook))
-                ==image_tag("Lock.png", class: "tagLogoLock tooltips", alt: "Private Notebook", title: "This notebook is private", tabindex: "0")
-              ==link_to_notebook(review.notebook)
+            td data-sort="#{render partial: GalleryConfig.slim.table_notebook_title_text_only, locals: { notebook: review.notebook }}"
+                ==render partial: GalleryConfig.slim.table_row_heading_label, locals: { notebook: review.notebook }
+                -if !review.notebook.public? && (@user.admin? || @user.can_edit?(review.notebook))
```

Comment:
Is the `(@user.admin? || @user.can_edit?(review.notebook))` check needed here? Looking at line 38 it seems if they don't meet these conditions they won't see the notebook anyways.